lib/ansible/modules/cloud/azure/azure_rm_acs.py

```diff
@@ -18,7 +18,7 @@
 version_added: "2.4"
 short_description: Manage an Azure Container Service Instance (ACS).
 description:
-    - Create, update and delete an Azure Container Service Instance (ACS).
+    - Create, update and delete an Azure Container Service Instance.
```

Comment:
Create, update and delete an Azure Container Service (ACS) instance.

Reply:
The term is "Azure Container Service", "instance" is not in the term, no need to capitalize it

Reply:
Thanks Yuwzho!